src/cuga/backend/cuga_graph/nodes/cuga_agent_core/policy/tool_approval_handler.py

```
@@ -221,6 +243,70 @@ async def check_and_create_approval_interrupt(
+        configurable: dict = config.get("configurable", {}) if isinstance(config, dict) else {}
+        skill_allowed_tools = configurable.get("skill_allowed_tools")
+        if skill_allowed_tools is None:
+            return None
+
+        try:
+            from cuga.backend.slash_commands.allowed_tools_enforcement import (
+                find_disallowed_calls,
+            )
+
+            disallowed = find_disallowed_calls(code, skill_allowed_tools)
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)